mmv1/products/securitycenterv2/ProjectMuteConfig.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ProjectMuteConfig'
+base_url: projects/{{project}}/locations/{{location}}/muteConfigs
+self_link: 'projects/{{project}}/locations/{{location}}/muteConfigs/{{mute_config_id}}'
+create_url: projects/{{project}}/locations/{{location}}/muteConfigs?muteConfigId={{mute_config_id}}
+update_verb: :PATCH
+update_mask: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/muteConfigs/{{mute_config_id}}'
+description: |
+  Mute Findings is a volume management feature in Security Command Center
+  that lets you manually or programmatically hide irrelevant findings,
+  and create filters to automatically silence existing and future
+  findings based on criteria you specify.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/projects.muteConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_project_mute_config_basic'
+    primary_resource_id: 'default'
+    skip_test: true
+    vars:
+      mute_config_id: 'my-config'
+    test_env_vars:
+      project: :PROJECT_NAME
+parameters:
+  - !ruby/object:Api::Type::String
+    name: project
```

Comment:
The build fail comes from this two project fields being included, this can be removed since it would already include the project field, the error:

`google-beta/services/securitycenterv2/resource_scc_v2_project_mute_config.go:124:4: duplicate key "project" in map literal`

Reply:
addressed